docs/contributing.rst

```diff
@@ -7,22 +7,32 @@ Contributing
 Development install
 -------------------
 
-**Note:** You will need :ref:`Node.js <https://nodejs.org/>` to build the extension package.
+**Note:** You will need `Node.js <https://nodejs.org/>`_ to build the extension package.
 
 The ``jlpm`` command is JupyterLab's pinned version of
-:ref:`yarn <https://yarnpkg.com/>` that is installed with JupyterLab. You may use
-:ref:`yarn <https://yarnpkg.com/>` or :ref:`npm <https://www.npmjs.com/>` in lieu of ``jlpm`` below.
+`yarn <https://yarnpkg.com/>`_ that is installed with JupyterLab. You may use
+`yarn <https://yarnpkg.com/>`_ or `npm <https://www.npmjs.com/>`_ in lieu of ``jlpm`` below.
 
 .. code-block:: bash
 
     # Clone the repo to your local environment
     git clone https://github.com/geojupyter/jupytergis.git
+
     # Change directory to the jupytergis directory
     cd jupytergis
+
+    # Create a virtual environment and activate it
+    python -m venv .venv
```

Comment:
Can I suggest also adding instructions using `micromamba`? Using a general purpose package manager makes it way easier to also install non-python packages like `yarn` and `qgis`:

```bash
micromamba create --name jupytergis_dev -c conda-forge yarn=3 "nodejs<22" jupyterlab pip qgis

micromamba activate jupytergis_dev
```

Reply:
Maybe yarn is required for the `jlpm run watch` script. `jlpm build` works fine for me w/o yarn installed separately, but `jlpm run watch` fails:

```
—————————————————————————————————————————————————————
   ✖  @jupytergis/jupytergis-lab:build                                                         
      /bin/sh: 1: yarn: not found                              
      /bin/sh: 1: yarn: not found                                                              
                                                                                                                                                                                               
                                                                                               
                                                                                               
—————————————————————————————————————————————————————
   ✖  @jupytergis/jupytergis-qgis:build                                                        
      /bin/sh: 1: yarn: not found               
      /bin/sh: 1: yarn: not found
```

Looks like this is controlled by `lerna.json`.

Reply:
874b4cb7ef45aae79ce0dffcd9fe24e56adae9e9

4660151014f89851314a96b5fca28e223b39a8b4

Reply:
> jlpm build works fine for me w/o yarn installed

Right! We can actually consider this a bug a replace [this](https://github.com/geojupyter/jupytergis/blob/32ae6a338d5ce37d1201e7cd2006629909f6ae76/lerna.json#L2) with `jlpm`, that would be less error prone than depending on the yarn version people have installed.